MSYS2/MinGW, CMake Error: failed to create symbolic link

`cmake.exe -E create_symlink glslang.exe glslangValidator.exe`
MSYS2/MinGW does not support symbolic links:
```
[422/422] Linking CXX executable StandAlone\glslang.exe
FAILED: StandAlone/glslang.exe 
cmd.exe /C "cd . && G:\MABS\msys64\mingw32\bin\ccache.exe  g++ -fstack-protector-strong -mtune=generic -O2 -pipe -mthreads -O3 -DNDEBUG -pipe -static-libgcc -fstack-protector-strong -static-libstdc++   -static -static-libgcc -static-libstdc++  -Wl,--no-undefined StandAlone/CMakeFiles/glslang-standalone.dir/StandAlone.cpp.obj -o StandAlone\glslang.exe -Wl,--out-implib,StandAlone\libglslang.dll.a -Wl,--major-image-version,0,--minor-image-version,0  glslang/libglslang.a  SPIRV/libSPIRV.a  glslang/libglslang-default-resource-limits.a  SPIRV/libSPVRemapper.a  -lpsapi  glslang/libMachineIndependent.a  OGLCompilersDLL/libOGLCompiler.a  glslang/OSDependent/Windows/libOSDependent.a  glslang/libGenericCodeGen.a  External/spirv-tools/source/opt/libSPIRV-Tools-opt.a  External/spirv-tools/source/libSPIRV-Tools.a  -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cmd.exe /C "cd /D G:\MABS\build\glslang-git\build-32bit\StandAlone && G:\MABS\msys64\mingw32\bin\cmake.exe -E create_symlink glslang.exe glslangValidator.exe""
CMake Error: failed to create symbolic link 'glslangValidator.exe': Dem Client fehlt ein erforderliches Recht. { Client lacks permission }

```
[logs.zip](https://github.com/m-ab-s/media-autobuild_suite/files/12100516/logs.zip)

Related to https://github.com/m-ab-s/media-autobuild_suite/issues/2482

--- a/StandAlone/CMakeLists.txt
+++ b/StandAlone/CMakeLists.txt
@@ -109,14 +109,18 @@ if(ENABLE_GLSLANG_INSTALL)
 
     # Create a symbolic link to glslang named glslangValidator for backwards compatibility
     set(legacy_glslang_name "glslangValidator${CMAKE_EXECUTABLE_SUFFIX}")
+    set(link_method create_symlink)
+    if (WIN32 OR MINGW)
+        set(link_method copy_if_different)
+    endif()
     add_custom_command(TARGET glslang-standalone
                        POST_BUILD
-                       COMMAND ${CMAKE_COMMAND} -E create_symlink $<TARGET_FILE_NAME:glslang-standalone> ${legacy_glslang_name}
+                       COMMAND ${CMAKE_COMMAND} -E ${link_method} $<TARGET_FILE_NAME:glslang-standalone> ${legacy_glslang_name}
                        WORKING_DIRECTORY $<TARGET_FILE_DIR:glslang-standalone>)
 
     # Create the same symlink at install time
     install(CODE "execute_process( \
-                      COMMAND ${CMAKE_COMMAND} -E create_symlink $<TARGET_FILE_NAME:glslang-standalone> ${legacy_glslang_name} \
+                      COMMAND ${CMAKE_COMMAND} -E ${link_method} $<TARGET_FILE_NAME:glslang-standalone> ${legacy_glslang_name} \
                       WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR})")
 
     if(ENABLE_SPVREMAPPER)

